modules/reitit-core/src/reitit/impl.cljc

```diff
@@ -155,3 +157,53 @@
 
 (defn strip-nils [m]
   (->> m (remove (comp nil? second)) (into {})))
+
+;;
+;; Path-parameters, see https://github.com/metosin/reitit/issues/75
+;;
+
+(defn url-encode [s]
+  (some-> s
+          #?(:clj  (URLEncoder/encode "UTF-8")
+             :cljs (js/encodeURIComponent))
+          (.replace "+" "%20")))
```

Comment:
This does the wrong thing in JavaScript:

```
cljs.user=> (.replace "aaa" "a" "b")
"baa"
```

OTOH it's only needed for Clojure, since `js/encodeURIComponent` encodes space ` ` with `%20` anyway. For clarity, I'd call `.replace` only for Clojure.

Reply:
awesome catch!